resources/plugins/skynetiads/skynetiads-config.lua

```diff
@@ -18,10 +18,21 @@ if dcsRetribution and SkynetIADS then
     local includeBlueInRadio = false
     local debugRED = false
     local debugBLUE = false
-    local actMobile = false
-    local actMobileMaxEmissionTime = 120
-    local actMobileMinimumScootDistance = 500
-    local actMobileMaximumScootDistance = 3000
+    local actMobile = false    -- did not change to actMobile_shorad for backwards compatibility
```

Comment:
Backwards compatibility is technically irrelevant in this case, as long as the plugin's options don't change (i.e. the mnemonic in plugin.json) there shouldn't be an issue. Just an FYI, no need to change it since the comment clears up the confusion 👍